### DISCUSSION TITLE
은은솔 1회차 블로그 피드백 요청 드립니다🙏

### DISCUSSION THREAD

**Discussion by euneunsol** [Orm-camp/blog-feedback#6] [동글동글 2기 피드백]
[[https://velog.io/@qls0915/미니프로젝트-회고](https://velog.io/@qls0915/%EB%AF%B8%EB%8B%88%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%9A%8C%EA%B3%A0)](url)

첫번째 미니 프로젝트 회고글입니다! 여러 날에 걸쳐 완성된 글이라 부족함이 많이 있어요. 
피드백 잘 부탁드려요!

**Comment by dayoungh** [comment]
TimeLine 구현을 해내게 된 과정을 조금 자세히 써주시면 더 좋을 것 같습니다 (물론 저도... 안 썼지만.......😂)
git/github을 이용해 어떤 방식으로 협업했는지, 구현한 화면을 캡쳐해서 넣은 부분이 좋다고 생각합니다! 세세한 부분까지 알 수 있어서 좋기도하고 
이미지가 있으니 글이 더 다채롭다고 느꼈습니다.

좋은 글 감사합니다👏👏👏👏

**Comment by juuyeoon** [comment]
은솔님, 임주연 입니다! 프로젝트 회고글 쓰시느라 고생 많으셨어요 🙌🏻
결과 화면 이미지가 첨부되어 있으니 작성하신 글들이 어떤 이야기인지 파악하는 데에 도움이 많이 되었습니다!
사용자의 사용성도 고민하게 되셨다니, 매우 멋진 시각이라고 생각합니다!
다만 위의 다영님 말씀처럼 어렵게 해내신 TimeLine 구현 과정에 대해서 조금 더 자세히 언급해 주셨더라면 하는 생각도 들었습니다.
어떤 과정을 거쳐 은솔님의 스킬이 향상되었는지 알 수 있는 대목이 될 것 같다는 느낌이 드네요!

**Comment by taeyoung0001** [comment]
오!! 깔끔하게 잘 쓰셨네요!
사용자의 편의성에 이미지로 설명되는 부분이 있으면 더 좋을거같아요!
글 읽기에 가독성도 너무 좋네요!
감사합니다!

**Comment by hyyun1113** [comment]
은솔님~ 블로그 글 잘읽었습니다. 
구체적으로 글을 적어주셔서 이해하기 쉬운 글이었습니다. 그리고 프로젝트를 하면서 느낀점이나 아쉬운점 등이 잘 기술되어있어 앞으로 더 많이 성장하는 개발자가 되실거라는 생각이 듭니다! 내용적인 부분은 위에 다른 분들이 다 말씀해 주셔서 저는 오타에 대해서만 말씀드리겠습니다.

* 프로젝트 기간 2024.08.26 
=> 2024.08.27 (발표일까지 포함하면 27일입니다!)
* 프로젝트개요의 마지막줄 '홈화면에서 task를 아이콘을~' 부분 
=> 홈화면에서 task아이콘을 
* 사용자편의성의 '양쪽 꺽새 표시에~'부분
=> 양쪽 꺽쇠 표시 (사실 꺽쇠또한 옳은 표기는 아니나 보통 꺽쇠라고 자주 쓰입니다)

블로그 글 너무 잘읽었습니다! 다음 블로그 글도 기대됩니다

**Comment by ggaong** [comment]
가독성 좋게 글을 작성하신 것 같습니당 😀
timeline 구현이 큰 도전이었다고 하셨는데 이 부분에 대해 좀 더 자세히 작성해주면 좋을 것 같다는 생각이 드네요!!🙌🙌

프로젝트 하시느라 고생 많으셨습니다!!
